react-app/src/library/services/vendor-implementations/plantronics/plantronics.ts

```diff
@@ -271,6 +313,7 @@ export default class PlantronicsService extends VendorImplementation {
       })
       .finally(() => {
         this.isConnecting = false;
+        this.$lastStatusUpdate.next('isConnecting')
```

Comment:
Seems weird to be setting `this.isConnecting = false;` and then emitting `isConnecting`?

Reply:
+1